common/workunit/workunit.cpp

```diff
@@ -10411,8 +10411,11 @@ IPropertyTree * CLocalWUGraph::getXGMMLTreeRaw() const
     return p->getPropTree("xgmml");
 }
 
+bool workunitGraphCacheEnabled = false;
```

Comment:
Should this default on in 8.4 and then off in 9.6 to help reduce the potential for unexpected regressions.  Then it could be tested on development system to verify the impact.